INSTALL_ISSUES.md

```diff
@@ -22,3 +22,25 @@ ___
 `ImportError: liblapack.so.3: Cannot open shared object file: No such file or directory`
 
 You need to install Blas and Lapack. See [LINUX\_INSTALL.md](LINUX_INSTALL.md) for details.
+
+___
+
+`ModuleNotFoundError: No module named 'turicreate'` with Jupyter Notebook
+This is for Ubuntu>= 17.10. If you're using another distro, your command will be slightly different according to your package manager.
+
+sudo apt update
+sudo apt remove virtualenv python jupyter
+sudo apt install -y libstdc++6 python-setuptools libgconf-2-4
+sudo apt install python-pip python3-pip
+sudo pip install virtualenv notebook jupyter
```

Comment:
You seem to be installing everything twice. Once for Python2 and once for Python3. I don't think this makes sense. Users should just install for which ever one they are using.

Reply:
Yes. Both for python2 and python3. I should have commented that inside the instructions. I found that pip was only working for python2 packages. So I installed pip3 also to run python3 packages.

Reply:
Should I update the INSTALL_ISSUES.md file according to your suggestion and pull a new request?